gamestonk_terminal/stocks/stocks_controller.py

```diff
@@ -233,14 +235,25 @@ def call_news(self, other_args: List[str]):
             default=5,
             help="Number of latest news being printed.",
         )
+        parser.add_argument(
+            "-s",
+            "--start",
+            action="store",
+            dest="n_start_date",
+            type=valid_date,
+            default=datetime.now() - timedelta(days=7),
+            help="The starting date (format YYYY-MM-DD) to search articles from",
+        )
         try:
             ns_parser = parse_known_args_and_warn(parser, other_args)
             if not ns_parser:
                 return
 
+            s_date = ns_parser.n_start_date.strftime("%Y-%m-%d")
             newsapi_view.news(
                 term=self.ticker,
                 num=ns_parser.n_num,
+                s_from=s_date,
```

Comment:
You can add here the line that you have above since it's only used here, there's no need to create an s_date variable :)

Reply:
Will do

Reply:
@Flodur871 should we merge what you have here? Or do you want to push this on this same PR?

Reply:
It's almost ready, so I think it would be better to push it here